tox.ini

```diff
@@ -689,7 +689,7 @@ allowlist_externals =
 setenv =
     NODE_PORT = {env:NODE_PORT:9082}
     GITHUB_CI = {env:GITHUB_CI:false}
-    PYTEST_MODULES = {env:PYTEST_MODULES:frontend network container_workload}
+    PYTEST_MODULES = {env:PYTEST_MODULES:frontend network container_workload local_node}
```

Comment:
A small, Local Node should be not part of k8s
it is already part of 
syft.test.integration workflow